app/controllers/grant_submissions/submissions_controller.rb

```diff
@@ -112,7 +114,7 @@ def set_grant
     end
 
     def submission_redirect(grant, submission)
-      if current_user == submission.applicant
+      if current_user == submission.submitter
```

Comment:
does this need to be changed to `submission.submitter || submission.applicant`? the `else` is looking at grant roles.

Reply:
This method is used in create and edit. Still needs to check whether `current_user == submission.applicant`, no?

Reply:
I am not sure what you are asking here. Is it that it should be different on create and edit. So create is submitter and applicants and edit is just applicants?